networkx/algorithms/simple_paths.py

```diff
@@ -257,7 +257,7 @@ def _empty_generator():
 
 
 def _all_simple_paths_graph(G, source, targets, cutoff):
-    visited = dict.fromkeys([source])
+    visited = dict.fromkeys([source], True)
```

Comment:
Would this be more readable? 
```suggestion
    visited = {source: True}
```